diesel_migrations/migrations_internals/src/migration.rs

```diff
@@ -29,6 +29,31 @@ impl<'a> fmt::Display for MigrationName<'a> {
     }
 }
 
+#[allow(missing_debug_implementations)]
```

Comment:
This is not used, right ?

Reply:
Frankly, I just do not know why this is and therefore copied from the [`Name`](https://github.com/sivakov512/diesel/blob/3fa64e41b2f8825b0e8be493977fa90bc60dd4bf/diesel_migrations/migrations_internals/src/migration.rs#L9) signature

Reply:
I meant the struct below sorry (was on phone)

Reply:
Struct below (MigrationFileName) used in public function [`file_name`](https://github.com/sivakov512/diesel/blob/3fa64e41b2f8825b0e8be493977fa90bc60dd4bf/diesel_migrations/migrations_internals/src/migration.rs#L39), that used [here](https://github.com/sivakov512/diesel/blob/3fa64e41b2f8825b0e8be493977fa90bc60dd4bf/diesel_migrations/migrations_internals/src/lib.rs#L335) and [here](https://github.com/sivakov512/diesel/blob/3fa64e41b2f8825b0e8be493977fa90bc60dd4bf/diesel_migrations/migrations_internals/src/lib.rs#L359)

Reply:
Oh è_é I'm blind...